### PR TITLE
fix(Signup): flag tests

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Fix tests in signup fail random, because the service of flag have a delay to obtain the data

### CODE REVIEW

**Comment by cbernat** [comment]
Is it too difficult just to mock de call? Instead of incrementing the timeout

**Comment by fcoronelmakingsense** [comment]
> Is it too difficult just to mock de call? Instead of incrementing the timeout

fixed

File: src/components/Signup/Signup.test.js [hunk 1]
```diff
@@ -153,11 +153,11 @@ describe('Signup', () => {
...
     // Assert
     await waitFor(() => {
-      setTimeout(() => {
-        expect(container.querySelector('#iti-1__item-io').nextElementSibling.id).toBe(
-          'iti-1__item-bn',
-        );
-      }, 1000);
+      expect(
+        container
+          .querySelector('[data-country-code="io"]')
```
Comment (cbernat) [inline src/components/Signup/Signup.test.js:158]: 😮

**Pull request merged**